editor/plugins/script_text_editor.cpp

```diff
@@ -1164,11 +1164,20 @@ void ScriptTextEditor::_update_connected_methods() {
 	// Add override icons to methods.
 	methods_found.clear();
 	for (int i = 0; i < functions.size(); i++) {
-		StringName name = StringName(functions[i].get_slice(":", 0));
+		String raw_name = functions[i].get_slice(":", 0);
+		StringName name = StringName(raw_name);
 		if (methods_found.has(name)) {
 			continue;
 		}
 
+		// Account for inner classes
+		if (raw_name.contains(".")) {
+			// Strip inner class name from the method, and always grab last since
+			// our inner class might be inside another inner class
+			PackedStringArray parts = raw_name.split(".");
+			name = StringName(parts[parts.size() - 1]);
+		}
```

Comment:
Can avoid the split by doing something like this?
```
int pos = rfind(".");
if (pos != -1) {
	name =  substr(pos + 1);
}
```

Reply:
I think they meant you can remove the contains check as well, since if it doesn't contain a period it will return -1